tools/depends/target/pythonmodule-pil/Makefile

```diff
@@ -1,45 +1,23 @@
 include ../../Makefile.include
-DEPS= ../../Makefile.include Makefile pillow-crosscompile.patch
-
-VERSION.TXT := $(CMAKE_SOURCE_DIR)/version.txt
-APP_NAME=$(shell awk '/APP_NAME/ {print tolower($$2)}' $(VERSION.TXT))
+DEPS= ../../Makefile.include Makefile
 
 # lib name, version
 LIBNAME=Pillow
-VERSION=7.1.2
+VERSION=8.1.0
 SOURCE=$(LIBNAME)-$(VERSION)
 ARCHIVE=$(SOURCE).tar.gz
 
-PYTHONPATH=$(PREFIX)/lib/python3.8/site-packages/
-PILPATH=$(PYTHONPATH)
-LDSHARED=$(CC) -shared
-
-# Clear pkg-config data, as it will pull from Native pkg-config incorrectly
-ifeq ($(CROSS_COMPILING), yes)
-export PKG_CONFIG_PATH=
-export PKG_CONFIG_LIBDIR=${PREFIX}/lib/pkgconfig
-export PKG_CONFIG_SYSROOT_DIR=${SDKROOT}
-endif
+BASE_URL=https://files.pythonhosted.org/packages/73/59/3192bb3bc554ccbd678bdb32993928cb566dccf32f65dac65ac7e89eb311
```

Comment:
todo: remove

Reply:
added the todo, will remove both package baseurl's when uploaded to mirrors. hint, with the new mirrors, i have no way to do this, so will need a mirror admin to do that side of things.